### PR TITLE
Update repo metadata for 0.1.1 upstream release

### PR DESCRIPTION
**Pull request by lukpueh**
**Fixes issue #**:

**Description of the changes being introduced by the pull request**:
- Update installation instructions in README.md
  - the transport should soon be available via apt
- Add an upstream CHANGELOG.md file

**Please verify and check that the pull request fulfills the following
requirements**:

- [ ] The code follows the [Code Style Guidelines](https://github.com/secure-systems-lab/code-style-guidelines#code-style-guidelines)
- [ ] Tests have been added for the bug fix or new feature
- [ ] Docs have been added for the bug fix or new feature

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,45 +8,27 @@ from public [*rebuilders*](https://salsa.debian.org/reproducible-builds/debian-r
...
-# Get sources
-git clone https://github.com/in-toto/apt-transport-in-toto.git
-# Install requirements
-pip install -r apt-transport-in-toto/requirements.txt
-# Install transport
-ln -s apt-transport-in-toto/intoto.py /usr/lib/apt/methods/intoto
-chmod 755 /usr/lib/apt/methods/intoto
 ```
-
+sudo apt install apt-transport-in-toto
```
Comment (lukpueh) [inline README.md:16]: ping @h01ger. Is it too early to say this?

**Comment by h01ger** [comment]
On Wed, Jan 27, 2021 at 05:18:51AM -0800, lukpueh wrote:
> @lukpueh commented on this pull request.
> +sudo apt install apt-transport-in-toto
> ping @h01ger. Is it too early to say this? 

yup, the package is still in NEW:

https://ftp-master.debian.org/new.html
https://ftp-master.debian.org/new/apt-transport-in-toto_0.1.0.html
https://ftp-master.debian.org/stat.html

-- 
cheers,
	Holger

 ⢀⣴⠾⠻⢶⣦⠀
 ⣾⠁⢠⠒⠀⣿⡁       holger@(debian|reproducible-builds|layer-acht).org
 ⢿⡄⠘⠷⠚⠋⠀ PGP fingerprint: B8BF 5413 7B09 D35C F026 FE9D 091A B856 069A AA1C
 ⠈⠳⣄

Nach wieviel Einzelfällen wird ein Einzelfall zum Normalfall?
(Jan Böhmermann)

**Comment by lukpueh** [comment]
Alright. Will revert. Thanks for the quick reply, @h01ger!

**Comment by h01ger** [comment]
On Wed, Jan 27, 2021 at 05:35:38AM -0800, lukpueh wrote:
> Alright. Will revert. Thanks for the quick reply, @h01ger!

well, it will work soon.


-- 
cheers,
	Holger

 ⢀⣴⠾⠻⢶⣦⠀
 ⣾⠁⢠⠒⠀⣿⡁       holger@(debian|reproducible-builds|layer-acht).org
 ⢿⡄⠘⠷⠚⠋⠀ PGP fingerprint: B8BF 5413 7B09 D35C F026 FE9D 091A B856 069A AA1C
 ⠈⠳⣄

The greatest danger in times of turbulence is not the turbulence;
it is to act with yesterdays logic. (Peter Drucker)

**Comment by lukpueh** [comment]
> well, it will work soon.

I can add it back. Or we add it in 0.1.2. I'm fine either way.

**Comment by h01ger** [comment]
On Wed, Jan 27, 2021 at 05:48:39AM -0800, lukpueh wrote:
> I can add it back. Or we add it in 0.1.2. I'm fine either way.

please add it back.


-- 
cheers,
	Holger

 ⢀⣴⠾⠻⢶⣦⠀
 ⣾⠁⢠⠒⠀⣿⡁       holger@(debian|reproducible-builds|layer-acht).org
 ⢿⡄⠘⠷⠚⠋⠀ PGP fingerprint: B8BF 5413 7B09 D35C F026 FE9D 091A B856 069A AA1C
 ⠈⠳⣄

we'll all die. make a difference while you can. disobey. smile.

**Comment by lukpueh** [comment]
> please add it back.

Done. I suggest I merge and release 0.1.1 here on GitHub and we finalize the downstream Debian 0.1.1-1 release in #29. @fepitre, d'accord?

**Comment by fepitre** [comment]
> > please add it back.
> 
> Done. I suggest I merge and release 0.1.1 here on GitHub and we finalize the downstream Debian 0.1.1-1 release in #29. @fepitre, d'accord?

Pas de problème! :)

**Pull request merged**